ci/scripts/alps_uenv.sh

```diff
@@ -321,6 +321,22 @@ launch_reframe() {
     # -n HelloWorldTestMPIOpenMP
 }
 # }}}
+# {{{ launch_reframe_1arg
+launch_reframe_1arg() {
+    export RFM_AUTODETECT_METHODS="cat /etc/xthostname,hostname"
+    export RFM_USE_LOGIN_SHELL=1
+    # export RFM_AUTODETECT_XTHOSTNAME=1
+    # reframe -V
+    echo "UENV=$UENV"
+    reframe -C ./config/cscs.py \
+        --report-junit=report.xml \
+        -c "$img" \
+        --system=$system \
+        -r
+    # -J reservation=all_nodes
+    # -n HelloWorldTestMPIOpenMP
```

Comment:
Let's remove completely the commented out sections?

Reply:
https://github.com/eth-cscs/cscs-reframe-tests/pull/220/commits/5dc0aacb9ea2ed939095f562436b5176bd60531b